crates/debugger_ui/src/debugger_panel_item.rs

```diff
@@ -807,6 +819,17 @@ impl Render for DebugPanelItem {
                                     .disabled(thread_status != ThreadStatus::Stopped)
                                     .tooltip(move |cx| Tooltip::text("Step out", cx)),
                             )
+                            .when(capabilities.supports_step_back.unwrap_or(false), |this| {
```

Comment:
Can we move before the step over? I think that makes a bit more sense based on how the icons look

Reply:
Good point! Will move it